plugin/src/main/java/org/screamingsandals/bedwars/special/listener/ThrowableFireballListener.java

```diff
@@ -76,9 +77,9 @@ public void onDamage(EntityDamageByEntityEvent event) {
             return;
         }
 
-        if (event.getDamager() instanceof Fireball) {
-            if (event.getDamager().hasMetadata(player.getUniqueId().toString())
-                    && event.getDamager().hasMetadata("fireball")) {
+        if (event.getHitEntity() instanceof Fireball) {
```

Comment:
`ProjectileHitEvent#getHitEntity()` doesn't exist in 1.9.4: https://helpch.at/docs/1.9.4/org/bukkit/event/entity/ProjectileHitEvent.html

Reply:
this is already removed